ui/nim-status-client.pro

```diff
@@ -81,354 +81,11 @@ qnx: target.path = /tmp/$${TARGET}/bin
-    onboarding/KeysMain.qml \
-    onboarding/Login.qml \
-    onboarding/Login/AccountList.qml \
-    onboarding/Login/AccountView.qml \
-    onboarding/Login/ConfirmAddExistingKeyModal.qml \
-    onboarding/Login/SelectAnotherAccountModal.qml \
-    onboarding/Login/qmldir \
-    onboarding/Login/samples/AccountsData.qml \
-    onboarding/Login/samples/qmldir \
-    onboarding/OnboardingMain.qml \
-    onboarding/img/browser-dark@2x.jpg \
-    onboarding/img/browser-dark@3x.jpg \
-    onboarding/img/browser@2x.jpg \
-    onboarding/img/browser@3x.jpg \
-    onboarding/img/chat-dark@2x.jpg \
-    onboarding/img/chat-dark@3x.jpg \
-    onboarding/img/chat@2x.jpg \
-    onboarding/img/chat@3x.jpg \
-    onboarding/img/key.png \
-    onboarding/img/key@2x.png \
-    onboarding/img/next.svg \
-    onboarding/img/wallet-dark@2x.jpg \
-    onboarding/img/wallet-dark@3x.jpg \
-    onboarding/img/wallet@2x.jpg \
-    onboarding/img/wallet@3x.jpg \
-    onboarding/qmldir \
-    shared/AccountSelector.qml \
-    shared/AddButton.qml \
-    shared/Address.qml \
-    shared/ContactsListAndSearch.qml \
-    shared/CropCornerRectangle.qml \
-    shared/DelegateModelGeneralized.qml \
-    shared/FormGroup.qml \
-    shared/GasSelectorButton.qml \
-    shared/IconButton.qml \
-    shared/ImageCropper.qml \
-    shared/ImageCropperModal.qml \
-    shared/Input.qml \
-    shared/LabelValueRow.qml \
-    shared/MacTrafficLights.qml \
-    shared/ModalPopup.qml \
-    shared/NotificationWindow.qml \
-    shared/PopupMenu.qml \
-    shared/Identicon.qml \
-    shared/AssetAndAmountInput.qml \
-    shared/CopyToClipBoardButton.qml \
-    shared/GasSelector.qml \
-    shared/GasValidator.qml \
-    shared/RoundedImage.qml \
-    shared/SearchBox.qml \
-    shared/SeedPhraseTextArea.qml \
-    shared/Select.qml \
-    shared/SendToContractWarning.qml \
-    shared/Separator.qml \
-    shared/SeparatorWithIcon.qml \
-    shared/SplitViewHandle.qml \
-    shared/StatusTabButton.qml \
-    shared/StyledButton.qml \
-    shared/RoundedIcon.qml \
-    shared/StyledText.qml \
-    shared/StyledTextArea.qml \
-    shared/StyledTextEdit.qml \
-    shared/StyledTextField.qml \
-    shared/SVGImage.qml \
-    shared/TextWithLabel.qml \
-    shared/ToastMessage.qml \
-    shared/TransactionPreview.qml \
-    shared/TransactionFormGroup.qml \
-    shared/TransactionStackView.qml \
-    shared/img/check.svg \
-    shared/img/close.svg \
-    shared/img/loading.png \
-    shared/img/loading.svg \
-    shared/img/status-logo.svg \
-    shared/qmldir \
-    shared/status/StatusChatInputTextFormationAction.qml \
-    shared/status/StatusEmojiSuggestionPopup.qml \
-    shared/status/StatusInputListPopup.qml \
-    shared/status/StatusRadioButtonRow.qml \
-    shared/status/StatusSettingsLineButton.qml \
-    sounds/ErrorSound.qml
+DISTFILES += $$files($$PWD/*.qml, true)
 
-DISTFILES += $$files("StatusQ/*", true)
+OTHER_FILES += $$files($$PWD/*.otf, true)
```

Comment:
I suggest to do just this:
`RESOURCES += resources.qrc`  (I personally do this every time)
we have this qrc automatically generated by go script when `make` is running.
your extensions do not cover all variations (please check `generate-rcc.go` for the full list.

Reply:
@pinebit - might be worth putting the two line .pro file changes in to a PR. I tested this out and it works great, but I also don't use Qt Creator very often (if ever), so I don't know what the implications would be.

Reply:
In which case, we can remove the .pro changes in this PR.

Reply:
I agree, we'd better separate changes as you suggested.
As for the .pro, AFAIK it is not used anywhere else in the build, except for updating the translations:
> 6. In that second terminal, run `lupdate nim-status-client.pro`
but, it should work even with the proposed changes. Given that we often forget to update this .pro when adding a new QML/js file, it is still better to use resources instead.

Reply:
FYI: I use QtCreator when editing UI, because QtCreator has much more powerful intellisense and code navigation. Even VS Code with QML plugins cannot match it in productivity.

Reply:
This change is actually for QtCreator users. I always use QtCreator and it's very convenient to have all files (incl. StatusQ, qmldir etc) listed and traceable there. I will create a new issue and PR for this!

Reply:
> FYI: I use QtCreator when editing UI, because QtCreator has much more powerful intellisense and code navigation. Even VS Code with QML plugins cannot match it in productivity.

@pinebit Very interesting! I seem to always have issues with errors preventing the Design view from working. Do you use this view or just the code editor?